src/main/java/stsjorbsmod/relics/GrimoireRelic.java

```diff
@@ -25,6 +25,12 @@ public void atBattleStart() {
         AbstractPlayer p = AbstractDungeon.player;
         this.flash();
         AbstractDungeon.actionManager.addToBottom(new RememberSpecificMemoryAction(p, PatienceMemory.STATIC.ID));
+        this.counter = 0;
+    }
+
+    @Override
+    public void atTurnStart() {
+        this.counter++;
```

Comment:
I'm not sure we want this to be incrementing with Quicksilver; I asked for feedback about the interaction in discord, we might want to change this reflect `AbstractDungeon.actionManager.turn` instead of maintaining its own count

Reply:
I have a change ready for this, but I'll wait for a response before pushing it.

Reply:
Thinking about this a bit more, I realize that Grimoire the relic and Grimoire the card have nothing to do with each other. I suppose using the same Grimoire power asset that just says, "at the start of turn 7, exhume grimoire" would be enough?

Reply:
Thoughts on making the new power to handle the self-exhume on snap and on turn 7?